docs/grammar/SolidityParser.g4

```diff
@@ -314,7 +314,7 @@ errorDefinition:
  * Using directive to bind library functions to types.
  * Can occur within contracts and libraries.
  */
-usingDirective: Using identifierPath For (Mul | typeName) Semicolon;
+usingDirective: Using (identifierPath | (LBrace identifierPath (Comma identifierPath)* RBrace) | Mul) For (Mul | typeName) Semicolon;
```

Comment:
`usingDirective` is missing from sourceUnit definitions i.e., this PR also enables `using * for T` at source unit level, right?

Reply:
Right, maybe the is an argument for not having this logic in the parser :)